ui/src/pages/Product.js

```diff
@@ -15,7 +15,6 @@
  *  specific language governing permissions and limitations
  *  under the License.
  */
-
 import { Fragment, useEffect, useRef } from 'react';
 import Header from '../components/products//Header';
```

Comment:
```suggestion
import Header from '../components/products/Header';
```
Is this correct?

Reply:
it's not correct, but the build seems to be resolving it somehow, updated the change